DOCS-178 Architecture Overview (#71)
hazelcast/cloud-docs: docs/modules/ROOT/pages/serverless-cluster.adoc

= Serverless Clusters

> :description: {hazelcast-cloud} Serverless is a fully-managed cloud service that offers a simple pay-as-you-go pricing model for clusters. Serverless clusters scale dynamically to provide the resources that your application needs, and you pay for the resources that your application consumes. You can start by creating a cluster for free, without a payment card.

Reviewer: ```suggestion
:description: {hazelcast-cloud} Serverless is a managed cloud service that offers a pay-as-you-go pricing model. Serverless clusters scale to provide the resources that your application needs. You pay only for the resources that your application consumes.
```

Author: Updated as requested.

:toc-levels: 3
:page-serverless: true

{description}

> Serverless means that Hazelcast manages the cloud infrastructure for you. Each Serverless cluster is an independent deployment of Hazelcast running in isolation on a Kubernetes container. This design guarantees resource isolation (memory and CPU), and prevents resource stealing, as well as providing isolated network access.

Reviewer: ```suggestion
Serverless means that Hazelcast manages the cloud infrastructure for you. Each Serverless cluster is an independent deployment of Hazelcast in a Kubernetes container. This design guarantees resource isolation, prevents resource stealing, and provides isolated network access.
```

Author: Updated as requested.
